### PR TITLE
build/configs/rtl8730e: Reduce KM4 PSRAM and CA32 ATF size

### PR DESCRIPTION
**Pull request by ZhenBei-Sin**
Changes Note:
Total free up 6MB
- KM4_BD_DRAM reduce from 1388KB to 1MB
- CA32 ATF reduce from 1.5MB to 1MB
- Remove the KM4_AUDIO_BUFFER 128KB (SRAM) is for VAD as no required to use

### CODE REVIEW

**Comment by ZhenBei-Sin** [comment]
Heap Info:
image

**Comment by ZhenBei-Sin** [comment]
[16Jan24_ATF1M_KM4PRSRAM1M_CA32XIP_Free6M.zip](https://github.com/Samsung/TizenRT/files/13945680/16Jan24_ATF1M_KM4PRSRAM1M_CA32XIP_Free6M.zip)

File: os/arch/arm/include/amebasmart/chip.h
Comment (ZhenBei-Sin) [inline os/arch/arm/include/amebasmart/chip.h:1]: Please help to check further how to fill in the decimal value for the AMEBASMART_DRAM_SIZE.
For example: 6.5MB

**Comment by edwakuwaku** [comment]
Dear @sunghan-chang and @kishore-sn  ,

This PR is to align the memory definition in several locations. In the meantime we are working on to release more RAM, please help to review and merge this stable version.

Thank you.

File: os/arch/arm/include/amebasmart/chip.h
Comment (sunghan-chang) [reply os/arch/arm/include/amebasmart/chip.h:1]: @kishore-sn Could you check this?

**Comment by sunghan-chang** [comment]
@ZhenBei-Sin @edwakuwaku Once the onboarding test is done on this, let me merge.
@jylee9613 Could you share the test result?

**Comment by jylee9613** [comment]
@ZhenBei-Sin @edwakuwaku I confirmed onboarding is working well. Thank you.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**